internal/testing/fakeopenai/cassettes/chat-parallel-tools.yaml

```diff
@@ -0,0 +1,135 @@
+
+---
+version: 2
+interactions:
+    - id: 0
+      request:
+        proto: HTTP/1.1
+        proto_major: 1
+        proto_minor: 1
+        content_length: -1
+        host: api.openai.com
+        body: |-
+            {
+              "messages": [
+                {
+                  "content": "Answer in up to 3 words: Which ocean contains Bouvet Island?",
```

Comment:
oops I will fix this as my prompt didn't trigger a tool call. I will re-record before merge time